README.md

```diff
@@ -50,7 +50,9 @@ Full list of options in `config.json`:
 | -------------------------------------   | --------- | ------------ | ---------------------------------------------------------------                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
-| project_id                              | String    | Yes          | BigQuery project                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| project_id                              | String    | Yes          | BigQuery project
+| gcs_bucket                              | String    | Yes          | Google Cloud Storage Bucket to use to stage files                                                             |
+| gcs_key_prefix                          | String    |              | Prefix to use for staged files in Google Cloud Storage                                                        |
```

Comment:
what are you using this for? do you put here a different path for qa/prod instead of a different bucket? I think it makes sense to add it anyway but just trying to figure out use cases

Reply:
It's to make it match the changes from: https://github.com/transferwise/pipelinewise/pull/901

It is an optional key which defaults to an empty string anyway.

Reply:
To be honest, I'm not sure what folks use it for but it existed on the Snowflake implementation so I ported it.

Reply:
got it, that makes sense